com.ibm.wala.core/src/test/java/com/ibm/wala/core/tests/callGraph/ReflectionTest.java

```diff
@@ -844,12 +844,12 @@ public void testForNameThrownExceptions()
     CallGraph cg = CallGraphTestUtil.buildZeroCFA(options, new AnalysisCacheImpl(), cha, false);
     IMethod mainMethod = entrypoints.iterator().next().getMethod();
     List<CGNode> mainCallees =
-        Iterator2List.toList(cg.getSuccNodes(cg.getNode(mainMethod, Everywhere.EVERYWHERE)));
+        Iterator2Collection.toList(cg.getSuccNodes(cg.getNode(mainMethod, Everywhere.EVERYWHERE)));
```

Comment:
Is this related to JDK 11?  Or some ECJ change?

Reply:
Presumably it was motivated by some new diagnostic. But this change certainly is compatible with Java 1.8. Therefore, I will pull it out into a separate, tiny pull request of its own that we can merge before the big Java 11 change.